### PR TITLE
PCI weak update fix for errata kernel

### PR DESCRIPTION
**Pull request by johnsongeorge-w**
@DivyaJyothiKasam @santoshx @prashantmhetar 
Review the changes and notify if any modification required. Currently test is in progress. 
Testing effort is big so wanted to consolidate the comments and verify the full test in final version.

### CODE REVIEW

**Review by prashantmhetar — APPROVED** [review]
Below patch changes looks good
RH 7.X: Fix missing link for weak-updates of all microsoft-hyperv drivers

File: LISISO/RPMS73/install.sh [hunk 1]
```diff
@@ -5,76 +5,17 @@
...
-elif [[ "$kernelver" =~ $regex16 ]] ; then
-	installbuildrpm 16
-elif [[ "$kernelver" =~ $regex17 ]] ; then
-	installbuildrpm 17
-elif [[ "$kernelver" =~ $regex18 ]] ; then
-        installbuildrpm 18
-elif [[ "$kernelver" =~ $regex19 ]] ; then
-        installbuildrpm 19
-elif [ "$kernelver" == "3.10.0-514.el7.x86_64" ] || [ "$kernelver" == "3.10.0-514.2.2.el7.x86_64" ] || [ "$kernelver" == "3.10.0-514.6.1.el7.x86_64" ] || [ "$kernelver" == "3.10.0-514.6.2.el7.x86_64" ] ;then
+if ! IsInstalledKernelOlderThanErrataKernel $errata_kerver_1; then
```
Comment (prashantmhetar) [inline LISISO/RPMS73/install.sh:14]: This needs to be corrected with removal of '!' condition. Checking the same with multiple errata kernels

Comment (johnsongeorge-w) [reply LISISO/RPMS73/install.sh:14]: This is required and it is implemented to club all kernel which has no conflicts to use the same LIS RPM.

**Review by prashantmhetar — APPROVED** [review]
Looks good

**Pull request merged**